xbmc/addons/RepositoryUpdater.cpp

```diff
@@ -170,7 +174,7 @@ void CRepositoryUpdater::OnTimeout()
   }
 
   CLog::Log(LOGDEBUG,"CRepositoryUpdater: running scheduled update");
-  CheckForUpdates();
+  CheckForUpdates(false, true);
```

Comment:
The second argument forces install to true. Should it use the `install` variable instead?